lib/granite/action/policies.rb

```diff
@@ -55,17 +55,17 @@ def allow_self
         end
       end
 
-      def try_perform!(*)
+      def try_perform!(*args, **kwargs)
```

Comment:
That looks weird...

Reply:
Yeah, but we can either do this or use `ruby2_keywords` to make it work in ruby-3 (or silence warnings in ruby-2.7.[0-1]). 
The proper way for such cases is `def try_perform!(...)`, but it works only on >=2.7.

Reply:
Oh didn't know that, I guess it's fine then.